src/globus_sdk/services/transfer/client.py

```diff
@@ -61,9 +69,12 @@ class TransferClient(client.BaseClient):
     **Filter Formatting**
 
     Several methods of ``TransferClient`` take a ``filter`` parameter which can be a
-    string or dict. When the filter given is a string, it is not modified. When it is a
-    dict, it is formatted according to the following rules.
+    string, dict, or list of strings/dicts. When the filter given is a string, it is
+    passed as a single unmodified param. When the filter given is a dict, it is
+    formatted according to the below rules into a single param. When the filter is a
+    list, each item of the list is parsed and passed as separate params.
```

Comment:
I have an idea for refactoring this part of the doc after this change to include this blob of text in the `operation_ls` and `task_list` docstrings, which I want to try out after this merges.

I think we can improve this further (because we can distinguish the two cases), but I'll wait until I can tinker with docstring manipulation to see what works.